substrate/client/tracing/proc-macro/src/lib.rs

```diff
@@ -109,18 +109,22 @@ pub fn prefix_logs_with(arg: TokenStream, item: TokenStream) -> TokenStream {
 	if arg.is_empty() {
 		return Error::new(
 			Span::call_site(),
-			"missing argument: name of the node. Example: sc_cli::prefix_logs_with(<expr>)",
+			"missing argument: name of the node. Example: #[prefix_logs_with(\"MyNode\")]",
 		)
 		.to_compile_error()
-		.into()
+		.into();
 	}
 
 	let name = syn::parse_macro_input!(arg as Expr);
 
-	let crate_name = match crate_name("sc-tracing") {
-		Ok(FoundCrate::Itself) => Ident::new("sc_tracing", Span::call_site()),
-		Ok(FoundCrate::Name(crate_name)) => Ident::new(&crate_name, Span::call_site()),
-		Err(e) => return Error::new(Span::call_site(), e).to_compile_error().into(),
+	let crate_name = match crate_name("polkadot-sdk") {
+		Ok(FoundCrate::Name(sdk_name)) => Ident::new(sdk_name.as_str(), Span::call_site()),
```

Comment:
Please check how this is used; path `sc-tracing::logging` makes sense, but `polkadot-sdk::logging` does not.

Reply:
Used `generate_access_from_frame_or_crate`